src/CycloneDX.Core/CycloneDX.Core.csproj

```diff
@@ -15,9 +15,12 @@
     <PackageReference Include="JsonSchema.Net" Version="3.3.2" />
     <PackageReference Include="protobuf-net" Version="3.2.26" />
     <PackageReference Include="protobuf-net.BuildTools" Version="3.2.12" PrivateAssets="all" IncludeAssets="runtime;build;native;contentfiles;analyzers;buildtransitive" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">

```

Comment:
Adding a comment to explain the 'why' could be beneficial here. 
Maybe refer to [link](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?pivots=dotnet-9-0)?

Reply:
have added a comment & link